### PR TITLE
[Assessor] Update wording

### PR DESCRIPTION
**Pull request by JeremieSamson**

### CODE REVIEW

File: src/Assessor/AssessorRequestCommand.php [hunk 1]
```diff
@@ -48,6 +48,7 @@ class AssessorRequestCommand
     private $firstName;
 
     /**
+     * @Assert\NotBlank(message="assessor.birth_name.not_blank", groups={"fill_personal_info"})
```
Comment (thomasbisignani) [inline src/Assessor/AssessorRequestCommand.php]: The message `Votre nom de naissance est requis.` is generic, can you use instead : `common.birth_name.not_blank` ?

Like this new PR : https://github.com/EnMarche/en-marche.fr/pull/3610/files#diff-8f79de8cc8dbed0031a212b6338a458eR26

Comment (thomasbisignani) [reply src/Assessor/AssessorRequestCommand.php]: Also, can you add the assert on the entity ?

Comment (JeremieSamson) [reply src/Assessor/AssessorRequestCommand.php]: done, thanks :ok_hand:

File: tests/Controller/EnMarche/AssessorControllerTest.php [hunk 2]
```diff
@@ -34,6 +34,7 @@ public function testAssessorRequest()
                 'gender' => 'male',
                 'firstName' => 'Ernestino',
                 'lastName' => 'Bonsoirini',
+                'birthName' => 'Boujourini',
```
Comment (illusionOfParadise) [inline tests/Controller/EnMarche/AssessorControllerTest.php]: Perhaps `Bonjourini`? 😄

File: translations/validators.fr.yml [hunk 3]
```diff
@@ -23,6 +23,10 @@ common.email.invalid: Ceci n'est pas une adresse e-mail valide.
 common.email.max_length: L'adresse e-mail est trop longue, 255 caractères maximum.
 common.email.repeated: Les adresses email ne correspondent pas.
 
+common.birth_name.note_blank: Votre nom de naissance est requis.
```
Comment (illusionOfParadise) [inline translations/validators.fr.yml]: What about `Le nom de naissance est requis`?

**Comment by thomasbisignani** [comment]
You should also remove the `nullable` behavior of this field : 

- in the `AssessorRequestCommand` DTO (setter / getter ) 
- In the `AssessorRequestFactory::createFromArray` method
- In the `AssessorRequest` entity (create static method, setter / getter, ORM annotation)

**Pull request merged**